ui/src/criteria/attribute.tsx

```diff
@@ -81,23 +141,149 @@ function AttributeEdit(props: AttributeEditProps) {
   const hintDisplayName = (hint: tanagra.EnumHintValue) =>
     hint.displayName || "Unknown Value";
 
-  const enumHintValues = underlay.entities
+  const attributeFilterHint = underlay.entities
     .find((g) => g.name === underlay.primaryEntity)
-    ?.attributes?.find((attribute) => attribute.name === props.data.attribute)
-    ?.attributeFilterHint?.enumHint?.enumHintValues;
+    ?.attributes?.find(
+      (attribute) => attribute.name === props.data.attribute
+    )?.attributeFilterHint;
+  const enumHintValues = attributeFilterHint?.enumHint?.enumHintValues;
+  const integerBoundsHint = attributeFilterHint?.integerBoundsHint;
 
-  const selectionIndex = (hint: tanagra.EnumHintValue) =>
-    props.data.selected.findIndex(
-      (row) => row.id === hint.attributeValue?.int64Val
-    );
+  console.log(underlay);
+  console.log(attributeFilterHint);
 
-  if (enumHintValues?.length === 0) {
+  if (!isValid(integerBoundsHint) && enumHintValues?.length === 0) {
```

Comment:
This check should be made consistent with the check at L163. i.e. just make one check above:

```
const hasValidIntegerBounds = isValid(integerBoundsHint?.min) && isValid(integerBoundsHint?.max);
```

Reply:
same as the issue for L163